CovertInfiltration/Src/CovertInfiltration/Classes/X2ActivityTemplate_Infiltration.uc

```diff
@@ -18,6 +18,8 @@ var config int ExpirationBaseTime;
 var config int ExpirationVariance;
 var config bool ExpirationNotBlocksCleanup; // Inverted, so that default is "block cleanup"
 
+delegate string GetRewardDetailStringFn(XComGameState_Reward RewardState);
```

Comment:
I think it's better to pass `XCGS_Activity` here, as right now you end up calling `GetInfiltrationTemplateFromReward` twice

Reply:
I'm adding XCGS_Activity but keeping XCGS_Reward, it's too much of a hassle to get the RewardState from an ActivityState, especially when there's multiple rewards.

Reply:
ok, sounds good